Contribute/get-started-setup-local.md

```diff
@@ -1,150 +1,217 @@
+   A copy of the Microsoft repository will be created within the account you selected, this will be your fork.
 
-## Choose a local folder
-Make a local folder to hold a copy of the repository locally. Some of the repositories can be large; up to 5 GB for azure-docs for example. Choose a location with available disk space.
+## Choose a location on your device to clone your fork
 
-1. Choose a folder name should be easy for you to remember and type. For example, consider a root folder `C:\docs\` or make a folder in your user profile directory `~/Documents/docs/`
+Now that you have created a fork of the Microsoft repository, you must clone your fork locally. Some of the documentation repositories can be large; for example, `azure-docs` is up to 25 GB. Make sure to choose a location with available disk space for cloning the forked repository.
+
+When cloning a repository, Git will automatically create a new directory, with the forked repository's name,  within your current directory. If you are working on multiple documentation repositories it might be helpful to store them together. For example, you might want to clone the `azure-docs`, `sql-docs`, and `visualstudio-docs` using a directory path such as `C:/Users/<Username>/Documents/Docs/<Repo-Name>`. With each repository being stored separately in its own directory (`<Repo-Name>`), but together under the same parent directory (`Docs`).
+
+> [!TIP]
+> [Windows Terminal](/windows/terminal/) with [PowerShell](/powershell/) supports a nearly identical command experience to that of Git Bash. You can even run Git Bash inside Windows Terminal.
+
+1. Launch Git Bash
+
+   :::image type="content" source="media/contribute-get-started-setup-local/gitbash-start.png" alt-text="Launch Git Bash from Windows search.":::
+
+   The default path that Git Bash starts in is the home directory (`~`) or `/c/Users/<Username>` on Windows. To determine the current directory, type `pwd`.
+
+2. Change directory (`cd`) to the directory that you want to create your parent directory in. Note that Git Bash uses the Linux convention of forward-slashes instead of back-slashes for directory paths.
 
    > [!IMPORTANT]
-   > Avoid choosing a local folder path that is nested inside of another git repository folder location. While it is acceptable to store the git cloned folders adjacent to each other, nesting git folders inside one another causes errors for the file tracking.
+   > Avoid choosing a local directory path that is nested inside of another Git repository. While it is acceptable to store the cloned repositories adjacent to each other, nesting repositories inside one another causes errors for the Git file tracking.
 
-2. Launch Git Bash
+   Consider a root directory:
 
-   ![Launch Git Bash](./media/contribute-get-started-setup-local/gitbash-start.png)
+   ```bash
+   cd C:/
```

Comment:
Does this work on macOs and Linux?

Reply:
Ahhh, good catch. It will not. I'll change this.

I think I'm going to change this to use the user directory (`cd ~`) instead of the root as well.